### DISCUSSION TITLE
Is it possible to register a plugin for a path with HttpMultipartResource?

### DISCUSSION THREAD

**Discussion by vesley** [SmingHub/Sming#2425] [Q&A] [answered]
https://github.com/SmingHub/Sming/blob/85d3214fff3d45eb65d508da5f1c8aee820662e7/Sming/Libraries/MultipartParser/samples/Conditional_Upload/app/application.cpp#L153

**Comment by slaff** [comment, marked as answer]
PR #2427 will make it possible. Once merged into `develop` you should be able to write:

```
server.paths.set("/upgrade", new HttpMultipartResource(fileUploadMapper, onUpload), 
                  new ResourceBasicAuth("realm", "username",  "password")
);
```
